packages/auth/demo/README.md

```diff
@@ -58,6 +59,20 @@ yarn run demo
 This will compile all the files needed to run Firebase Auth, and start a Firebase server locally at
 [http://localhost:5000](http://localhost:5000).
 
+The demo opens a page like this:
+
+![image](https://user-images.githubusercontent.com/35932340/153662957-41ba6a82-ea15-4084-ad3a-9fd41083efd3.png)
+
+
+This is a developer view of all the supported auth flows. Make sure that the auth flow you are testing is already enabled in your firebase project.
+For example, if you are testing “Sign up with email/password”, your project should allow email/password as a provider. 
+If not, you will see an “auth/operation-not-allowed” error message. 
+
+You can check the enabled providers on the firebase console.
+
+![image](https://user-images.githubusercontent.com/35932340/153662750-c0faf417-07b4-4f0e-93ab-5e0b82e3c793.png)
+
+
 ## Running against Auth Emulator
 
 The demo page by default runs against the actual Auth Backend. To run against the Auth Emulator with mocked endpoints, do the following:
```

Comment:
Can we add a pre-requisite for this step - 
You need to have Java install.

Reply:
Java prereq is specified in the top level readme. I linked to it in this doc - https://github.com/firebase/firebase-js-sdk#prerequisites